client/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		"baseUrl": ".",
 		"module": "commonjs",
-		"target": "es5",
+		"target": "es6",
```

Comment:
This was an odd one. We were targeting es6 in our src project but not our client project. This led to issues when having types inherit from each other across projects.